# goal
bool shouldArm
---
# result
bool result
---
# feedback
bool isArmed